.coderabbit.yaml

```
@@ -1,4 +1,9 @@
 reviews:
+  auto_review:
+    enabled: true
+    drafts: true
+    labels:
+      - "Ready for Review"
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)